docs/sources/logql/template_functions.md

```diff
@@ -635,6 +635,45 @@ Examples:
 {{ .foo | toDateInZone "2006-01-02T15:04:05.999999999Z" "UTC" | unixEpochNanos }}
 ```
 
+## toUnixEpoch
+
+Converts the string epoch to the number of seconds elapsed since January 1, 1970 UTC.
+
+Signature: `toUnixEpoch(epoch string) int64`
+
+Examples:
+
+Consider the following log line `{"from": "1679577215","to":"1679587215","message":"some message"}`. To print the `from` field as human readable add the following at the end of the LogQL query:
+```logql
+... | json | line_format `from="{{date "2006-01-02" (toUnixEpoch .from)}}"`
+```
+
+## toUnixEpochMillis
+
+Converts the string epoch to the number of milliseconds elapsed since January 1, 1970 UTC.
+
+Signature: `unixEpochMillis(epoch string) int64`
+
+Examples:
+
+Consider the following log line `{"from": "1257894000000","to":"1257894004000","message":"some message"}`. To print the `from` field as human readable add the following at the end of the LogQL query:
+```logql
+... | json | line_format `from="{{date "2006-01-02" (toUnixEpochMillis .from)}}"`
+```
+
+## toUnixEpochNanos
+
+Converts the string epoch to the number of nanoseconds elapsed since January 1, 1970 UTC.
+
+Signature: `toUnixEpochNanos(epoch string) int64`
```

Comment:
I think this signature is incorrect? should return a time.Time?

Reply:
This is replace with another function now.